library/src-bootstrapped/scala/quoted/Expr.scala

```diff
@@ -43,7 +58,7 @@ object Expr {
   def ofSeq[T](xs: Seq[Expr[T]])(using Type[T])(using Quotes): Expr[Seq[T]] =
     Varargs(xs)
```

Comment:
That won't work, the name of the method is not to distinguish the arguments but the target type. `Expr(seq)` should this return a `Expr[Seq[T]]` or a `Expr[List[T]]`? Also, `Exprs` might be a better place for it.

Let's keep this for a follow-up PR.

Reply:
Maybe we can only have `Value.apply` and `Value.unapply`. This would imply that `Expr(v)` will not work anymore.

Reply:
This part of the API clearly needs a deeper analysis. I will try it in a separate PR and we can keep this one only to remove the `Liftable`/`Unliftable`.

Reply:
`Values` seems to be a much better name as it captures much better the semantics of the operation in the names. The only downside is that it would affect 99% of the macros.

Reply:
> Maybe we can only have `Value.apply` and `Value.unapply`. This would imply that `Expr(v)` will not work anymore.

It seems to be `Expr(v)` is better than `Value.apply`, as we are constructing a value of `Expr[T]`. Could you please elaborate?

Reply:
To avoid confusions we need to have `Expr.apply`/`Expr.unapply` or `Value.apply`/`Value.unapply`.

Would you understand if you relate the `Some` concept if your code would be like the following?
```scala
Some(2) match
  case Just(n) =>
// or
Just(2) match
  case Some(n) =>
```

I would find it extremely confusing and would probably assume they are different concepts. Even though logically they are the dual of eachother.

Reply:
There is a difference here: for `Tree`s, we expect there are different cases correspond to different syntactic forms, but for `Option` we don't have the expectation.

Reply:
That would be true for trees but here we are on the `Expr` abstraction which hides those details. In the expression world you don't distinguish `'{1}`, '{{1}}`, '{{{1}}}`, ... . Those are only differences that can be seen on `Tree`s.